tests/test_player_fibonacci.py

```diff
@@ -0,0 +1,95 @@
+from unittest import TestCase
+from table import Table
+from bet import Bet
+from players.player_fibonacci import PlayerFibonacci
+
+
+class TestPlayerFibonacci(TestCase):
+    def setUp(self):
+        self.table = Table()
+        self.player_fibonacci = PlayerFibonacci(self.table)
+        self.bet = Bet(1, self.player_fibonacci.outcome)
+
+    def test_win_resets_recent_value(self):
```

Comment:
test name need to be renamed. e.g `test_recent_value_is_reset_after_win`

Reply:
Done.